ui/app/components/sidebar/nav/cluster.hbs

```diff
@@ -13,6 +13,9 @@
     @text="Secrets engines"
     data-test-sidebar-nav-link="Secrets engines"
   />
+  {{#if this.version.isEnterprise}}
```

Comment:
Good call including this conditional. There's a note in the overview ticket we may change this if design decides to create a promotional "upgrade to enterprise" view for non-enterprise users